cscope.py

```diff
@@ -41,6 +42,14 @@ def get_setting(key, default=None, view=None):
         pass
     return get_settings().get(key, default)
 
+def update_result_selection(view, selection_index, selection, update_sel=True):
```

Comment:
I'm confused on what this is doing here too. Can you please help me understand what this is doing that it wasn't already doing? Also, I suspect that this will not work with Sublime Text 2. Thus far, while I'm not actively supporting Sublime Text 2, I've been making sure I at least don't break it with changes. Can you verify that this will at least not cause errors in ST2?

Reply:
I keep results and currently selected results in view settings of every Cscope results. 
I checked on ST2 and fixed scrolling to result and disabled class for mouse support.
But. The master branch wasn't working on ST2 :) Minor problems:
- trailing comma in settings
- ```ValueError: zero length fields name in format``` cause of lack of indices in the format specs